developer-docs-site/docs/concepts/coin-and-token/mint-minting-tool.md

```diff
@@ -0,0 +1,91 @@
+
+- Download the assets and decompress the compressed file
+
+`easel` works with the images and metadata generated by `HashLips` out of the box.
+
+The metadata format we need is as simple as the following:
+
+`{ "attributes": [ { "trait_type": "Background", "value": "Black" }, { "trait_type": "Eyeball", "value": "Red" } ] }`
+
+Note: other fields from `HashLips` are ignored.
+
+Each NFT should have a separate metadata file in JSON format. All NFT metadata files need to be put under a folder named `json`. All NFT asset files need to be put under another folder named `images`. `json` and `images` need to be under the same paren folder. See below example:
+
+<pre>
+<code>
+nft_assets/
+├─ images/
+│ ├─ 1.png
+│ ├─ 2.png
+│ ├─ 3.png
+│ ├─ ...
+├─ json/
+│ ├─ 1.json
+│ ├─ 2.json
+│ ├─ ...
+</code>
+</pre>
+
+Each pair of image and metadata file represent an NFT. E.g. `1.png` and `1.json` in the above example are the asset and metadata files of the first NFT.
+
+## Create an NFT project
+
+- Create a project called `test` with command `easel init --name test --asset-path <decompressed-asset-folder>`
+- Change your current directory to `test` with the command `cd test`
+
+## Compile and publish the smart contract
+
+- Run command `easel publish-contract --profile nft_mint`
+
+## Fund the storage service
+
+We would like to upload assets to Arweave and we need to purchase the credits. Bundlr makes the process easier. We need to fund the Bundlr service before we can upload assets.
+
+- Run `easel fund --profile nft_mint --amount 10000000` to fund Bundlr
+- Check the balance `easel balance --profile nft_mint`
+
+## Upload the assets
+
+- Change current folder to the project folder `test`
+- Run command to upload assets `easel upload --profile nft_mint`
+
+## **Run the minting website**
+
+Now you can run the minting site and start minting NFTs.
+
+- Download the minting site code [https://storage.googleapis.com/nft-mint/minting-site.zip](https://storage.googleapis.com/nft-mint/minting-site.zip)
+- Decompress the code
+- Change the current directory to the decompressed folder and run `npm install`
+- Run `REACT_APP_MINTING_CONTRACT="0xc87947527858ce6fd0eb89e1a15ff8461d8c894b86bd7c572306602aa58c8cc5" REACT_APP_NODE_URL="[https://fullnode.testnet.aptoslabs.com](https://fullnode.testnet.aptoslabs.com/)" npm run start`. Make sure `REACT_APP_MINTING_CONTRACT` points to the correct resource account address. The account address needs to be prefixed with `0x`. `REACT_APP_NODE_URL` needs to point to the correct node URL. For testnet, the url is [`https://fullnode.testnet.aptoslabs.com`](https://fullnode.testnet.aptoslabs.com/). For mainnet, the url is [`https://fullnode.mainnet.aptoslabs.com`](https://fullnode.testnet.aptoslabs.com/). And then, you can visit the minting site, by visiting http://localhost:3000
```

Comment:
In these instructions, we should go back and tell people to note the address when they create the account, or it will be confusing, unless this is the address of the testnet account.